### DISCUSSION TITLE
My god, it's full of stars!

### DISCUSSION THREAD

**Discussion by gmischler** [py-pdf/fpdf2#1232] [Announcements]
It can only have happened a few days ago, but I just noticed that this project has now been "starred" ⭐  a thousand times.

Friends, delevopers, contributors - congratulations!  🎆 🌟 🍾

**Comment by Lucas-C** [comment]
Thank you for mentioning it @gmischler! 👍

And thanks to all https://github.com/py-pdf/fpdf2 for helping to reach this milestone 🙏

![n9Wy5fF](https://github.com/user-attachments/assets/497b498a-2e7f-4c4a-9e9f-761f347a6e16)
